src/panels/config/voice-assistants/cloud-google-pref.ts

```diff
@@ -0,0 +1,352 @@
+
+    return html`
+      <ha-card outlined>
+        <h1 class="card-header">
+          <img
+            alt=""
+            src=${brandsUrl({
+              domain: "google_assistant",
+              type: "icon",
+              darkOptimized: this.hass.themes?.darkMode,
+            })}
+            referrerpolicy="no-referrer"
+          />${this.hass.localize("ui.panel.config.cloud.account.google.title")}
+        </h1>
+        <div class="header-actions">
+          <a
+            href="https://www.nabucasa.com/config/google_assistant/"
+            target="_blank"
+            rel="noreferrer"
+            class="icon-link"
+          >
+            <ha-icon-button
+              .label=${this.hass.localize(
+                "ui.panel.config.cloud.account.google.link_learn_how_it_works"
+              )}
+              .path=${mdiHelpCircle}
+            ></ha-icon-button>
+          </a>
+          <ha-switch
+            .checked=${google_enabled}
+            @change=${this._enabledToggleChanged}
+          ></ha-switch>
+        </div>
+        <div class="card-content">
+          <p>
+            ${this.hass.localize("ui.panel.config.cloud.account.google.info")}
+          </p>
+          ${manualConfig
+            ? html`<ha-alert alert-type="warning">
+                ${this.hass.localize(
+                  "ui.panel.config.cloud.account.google.manual_config"
+                )}
+              </ha-alert>`
+            : ""}
+          ${!google_enabled
+            ? ""
+            : html`${!google_registered
+                  ? html`
+                      <ha-alert
+                        .title=${this.hass.localize(
+                          "ui.panel.config.cloud.account.google.not_configured_title"
+                        )}
+                      >
+                        ${this.hass.localize(
+                          "ui.panel.config.cloud.account.google.not_configured_text"
+                        )}
+
+                        <ul>
+                          <li>
+                            <a
+                              href="https://assistant.google.com/services/a/uid/00000091fd5fb875?hl=en-US"
+                              target="_blank"
+                              rel="noreferrer"
+                            >
+                              ${this.hass.localize(
+                                "ui.panel.config.cloud.account.google.enable_ha_skill"
+                              )}
+                            </a>
+                          </li>
+                          <li>
+                            <a
+                              href="https://www.nabucasa.com/config/google_assistant/"
+                              target="_blank"
+                              rel="noreferrer"
+                            >
+                              ${this.hass.localize(
+                                "ui.panel.config.cloud.account.google.config_documentation"
```

Comment:
I opted not to, they will be loaded anyway now, as translation loading is per panel (config in this case) and it will safe a lot of new translations... 🤷‍♂️